editor/editor_spin_slider.cpp

```diff
@@ -461,3 +461,8 @@ EditorSpinSlider::EditorSpinSlider() {
 	read_only = false;
 	use_custom_label_color = false;
 }
+EditorSpinSlider::~EditorSpinSlider() {
+	if (grabbing_spinner) {
```

Comment:
this should probably better be in NOTIFICATION_EXIT_TREE, other than that it' s fine.